src/systems/physics/Physics.cc

```diff
@@ -1536,56 +1730,70 @@ void PhysicsPrivate::UpdateSim(EntityComponentManager &_ecm)
           components::Pose *_pose,
           const components::ParentEntity *_parent)->bool
       {
-        // If parent is static, don't process pose changes as periodic
+        // If model is static, don't process pose changes as periodic
```

Comment:
The entity inside this `Each` call should be a link though, and I believe links shouldn't have static components, right?

Reply:
fixed. a71a3d2e2e65bab10243987f91ed238a1329f7ab